### DISCUSSION TITLE
Allow members to view their templates without being workspace admins

### DISCUSSION THREAD

**Discussion by yezarela** [outline/outline#7153] [Ideas]
Currently, users with role editor can create template by creating a document -> templatize. But after editing the template, there is no way back the user can navigate to their created templates and edit from there, the only way is to direcly go to the template URL.

![image](https://github.com/outline/outline/assets/11919795/c76443b6-d41a-4f56-8011-9b7c4423edad)

![image](https://github.com/outline/outline/assets/11919795/d761d272-a01a-4fb8-a2a2-c13395932c5e)

It would be great if users with role editor can view their templates without necessity of being workspace admins, so they can edit their created templates from there

**Comment by tommoor** [comment]
This is being worked on here: https://github.com/outline/outline/pull/7150

**Reply by yezarela** [reply]
Thanks, glad it's being worked. Closing this

**Comment by wojtishek** [comment]
Sorry to reopen this conversation, but do you have any idea when this feature will be released?

**Comment by Codinator007** [comment]
Hi, i want to reopen this issue because i dont understand the workflow for my workspace users.
I tried to understand what you did here, but didnt find the solution  https://github.com/outline/outline/pull/7150

I think Users (Editors, non-admins)should have the ability to create Templates and to edit them later.
Am I missing something here? Because based on my tests, the user can create templates and as an admin I can see them but the user itself doesnt have any chance to edit them later right (Settings -> Templates disabled)?

It would be great, if the user who creates a template does than have access to Settings -> Templates and can see his Templates right?

Thank you in advance.
